pallets/compliance-manager/src/lib.rs

```diff
@@ -732,6 +766,34 @@ impl<T: Trait> Module<T> {
         }
         asset_rules_with_results
     }
+
+    fn verify_rules_complexity(
+        asset_rules: &Vec<AssetTransferRule>,
+        default_issuer_count: usize,
+    ) -> DispatchResult {
+        let mut complexity = 0usize;
+        for asset_rule in asset_rules {
+            for rule in asset_rule
+                .sender_rules
+                .iter()
+                .chain(asset_rule.receiver_rules.iter())
+            {
+                let (claims, issuers) = rule.complexity();
+                if issuers == 0 {
+                    complexity =
+                        complexity.saturating_add(claims.saturating_mul(default_issuer_count));
+                } else {
+                    complexity = complexity.saturating_add(claims.saturating_mul(issuers));
+                }
+            }
+        }
+        if let Ok(complexity_u32) = u32::try_from(complexity) {
+            if complexity_u32 <= T::MaxRuleComplexity::get() {
+                return Ok(());
```

Comment:
can be `Ok(())`

Reply:
The return type of this if block is supposed to be `()`. Converting the above line to `Ok(())` will cause a compiler error. 

The `return` keyword specifically instructs to return from the function (not just from the block).